helmfile/charts/prometheus-alerts/templates/alerts/packets-dropped.yaml

```
@@ -0,0 +1,42 @@
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.networkpolicies }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-alerts.fullname" .) "packets-dropped" | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ template "prometheus-alerts.name" . }}
+    {{- include "prometheus-alerts.labels" . | nindent 4 }}
+    {{- if .Values.defaultRules.alertLabels }}
+    {{- toYaml .Values.defaultRules.alertLabels | indent 4 }}
+    {{- end }}
+  {{- if .Values.defaultRules.annotations }}
+  annotations:
+    {{- toYaml .Values.defaultRules.annotations | indent 4 }}
```

```suggestion
    {{- toYaml .Values.defaultRules.alertLabels | nindent 4 }}
    {{- end }}
  {{- if .Values.defaultRules.annotations }}
  annotations:
    {{- toYaml .Values.defaultRules.annotations | nindent 4 }}
```

I assume these needs to be changed then as well.